docs/run-platform-server/configuring.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Configuring
+sidebar_position: 30
+---
+
+import { Alert } from "@site/src/components/Alert";
+import { CodeExample } from "@site/src/components/CodeExample";
+
+Pre-requisites:
+
+- [Installation](./installing.mdx) is completed on host machine, it should have two executables, `stellar-horizon` and `stellar-core`. Per the installation instructions, `stellar-core` should be included in your host o/s PATH as these configurations will result in Horizon searching for `stellar-core` to execute.
+- [Prepare the Database if first time](#prepare-the-database-first-time)
+
+You are now ready to setup configuration parameters to perform three important, distinct roles:
+
+- **serving requests** a regular web-based HTTP API,
+- **ingesting ledgers** from core nodes of a Stellar network to keep its world-view up to date, and
+- **transaction submission** for sending transactions to the Stellar network.
+
+You choose from one of two following deployment modes to perform the roles and proceed with configuration parameters to that means. A note on configuration parameters, the same parameter can be provided either as command line argument or an environment variable.
+
+## Single Instance Deployment
+
+You run `stellar-horizon` in a single o/s process and it will perform all three roles at same time.
+
+| flag               | envvar         | example                              |
+| ------------------ | -------------- | ------------------------------------ |
+| `--db-url`         | `DATABASE_URL` | postgres://localhost/horizon_testnet |
+| `--network`        | `NETWORK`      | pubnet                               |
+| `--log-level`      | `LOG_LEVEL`    | info                                 |
+| `--tx-sub-enabled` | TX_SUB_ENABLED | true                                 |
+
+## Multiple Instance Deployment
+
+You run multiple instances of `stellar-horizon` each performs a subset of the roles.
+
+### Ingestion Role Instance
+
+You need to allocate one instance to perform ongoing ingestion to capture network activity. We recommend limiting the amount of network activity to retain in the db on a sliding window of the last 30 days, we include `--history-retention-count 518400` which is number of ledgers in 30 days. **TODO - make --history-retention-count default to 518400, don't specify here anymore. make --captive-core-use-db=true by default**
+
+| flag | envvar | example |
+| --- | --- | --- |
+| `--db-url` | `DATABASE_URL` | postgres://localhost/horizon_testnet |
+| `--history-retention-count` | `HISTORY_RETENTION_COUNT` | 518400 |
+| `--network` | `NETWORK` | pubnet |
+| `--log-level` | `LOG_LEVEL` | info |
+
+### API Role Instance
+
+You can run none or multiple instances to serve read only API requests. **TODO - create --tx-sub-enabled flag, default=false. Should return 405 on POSTs to /tx if enabled. this allows skipping need to define --stellar-core-url on api servers since instance doesn't need to be aware of core.**
```

Comment:
```suggestion
You can run zero or more instances to serve read-only API requests. **TODO - create --tx-sub-enabled flag, default=false. Should return 405 on POSTs to /tx if enabled. this allows skipping need to define --stellar-core-url on api servers since instance doesn't need to be aware of core.**
```